test/test-lib/fixtures.ts

```diff
@@ -14,7 +155,34 @@ export const setDefaultFixtures = (
 
 export type FixtureData = Dictionary<Dictionary<any>>;
 
-export const load = async (): Promise<FixtureData> => {
-	const fixtures = { ...defaultFixtures };
-	return Bluebird.props(_.mapValues(fixtures, fx => Bluebird.props(fx)));
+export const load = async (fixtureName?: string): Promise<FixtureData> => {
```

Comment:
I'd much rather:

```suggestion
export const load = async (fixtureName: string): Promise<FixtureData> => {
```

and then exporting a separate function:

```js
export const loadDefault = async (): Promise<FixtureData> => {
```

These two internally could either use this one (with the nullable argument), or `load(...)` could use `loadDefault()`.

Reply:
I'd rather not do this as having a separate `loadDefault` implies that `load` wouldn't those defaults

Reply:
Makes sense. My concern is that for pretty much all uses of `load()`, the argument will be needed and the exported API merely suggests it -- does not enforce it. Maybe add a doc comment that not passing an argument will only load the default fixtures? VScode is pretty good making doc comments visible.

Reply:
Done